client/go/outline/electron/main.go

```diff
@@ -88,6 +91,7 @@ func main() {
 	args.logLevel = flag.String("logLevel", "info", "Logging level: debug|info|warn|error|none")
 	args.dnsFallback = flag.Bool("dnsFallback", false, "Enable DNS fallback over TCP (overrides the UDP handler).")
 	args.checkConnectivity = flag.Bool("checkConnectivity", false, "Check the proxy TCP and UDP connectivity and exit.")
+	args.fetchConfig = flag.String("fetchConfig", "", "The HTTPS URL of a dynamic key to fetch")
```

Comment:
Add details that it exits and does nothing else? Similar to `checkConnectivity`.

Reply:
Updated